gdb-7.8/gdb/breakpoint.c

```diff
@@ -303,6 +303,9 @@ static struct breakpoint_ops longjmp_breakpoint_ops;
    breakpoints.  */
 struct breakpoint_ops bkpt_breakpoint_ops;
 
+/* The breapoint_ops structure to be used in HSA library breakpoint */
+struct breakpoint_ops hsa_breakpoint_ops;
```

Comment:
I am not sure if hsa_breakpoint_ops is appropriate since these functions are not called for the case of creating breakpoints, i.e. it is only for the trigger. 
I think a better name may be "gpu_breakpoint_trigger_ops" ?